drivers/scheduler/k8s/k8s.go

```diff
@@ -3019,6 +2993,49 @@ func (k *K8s) addLabelsToPVC(pvc *v1.PersistentVolumeClaim, labels map[string]st
 	}
 }
 
+// CreateAutopilotRule creates the AutopilotRule object
+func (k *K8s) CreateAutopilotRule(apRule apapi.AutopilotRule) (*apapi.AutopilotRule, error) {
+	t := func() (interface{}, bool, error) {
+		apRule.Labels = defaultTorpedoLabel
+		aRule, err := k8sAutopilot.CreateAutopilotRule(&apRule)
+		if errors.IsAlreadyExists(err) {
```

Comment:
fix as we discussed offline

Reply:
just for context, what is the fix required here?

Reply:
return error should be inside if errors.IsAlreadyExists(err) {

Reply:
move `if err != nil {` above if `errors.IsAlreadyExists(err) {` line

Reply:
@mborodin-px  looks like this is still not done

Reply:
@harsh-px it is there: